src/frontend/apps/standalone_site/src/features/Playlist/features/UpdatePlaylist/components/UpdatePlaylistPage.spec.tsx

```diff
@@ -11,6 +11,7 @@ import { UpdatePlaylistPage } from './UpdatePlaylistPage';
 const deferredPlaylist = new Deferred<Playlist>();
 const playlist = {
   id: 'some-id',
+  retention_duration: null,
```

Comment:
You should add a test to check the retention_duration is displayed and update correctly.